docs/_docs/best-practices/general/general.md

```diff
@@ -28,6 +29,157 @@ Discover essential FinOps best practices to optimize cost efficiency and governa
 
 ---
 
+## Resource inventory summary
+
+### Query: Count of all resources
```

Comment:
Is this really necessary? It feels too simple to document. What scenario does this actually address? Fwiw, I think we have a larger effort to document scenarios across the entire BPL, but it's fine if we don't do that right now.